app/helpers/application_helper.rb

```diff
@@ -14,4 +14,12 @@ def state(document)
 
     content_tag(:span, state, class: classes).html_safe
   end
+
+  def pop_up_warning_for_publishing(document)
+    if document.update_type == 'minor'
+      "You are about to publish a minor edit. Continue?"
+    else
+      "Publishing will email subscribers to #{current_format.title.pluralize}. Continue?"
+    end
+  end
```

Comment:
I think this might be better placed in the newly created `PublishWarningPresenter` as it seems to fit with the intent of that presenter. What do you think?